easybuild/tools/filetools.py

```diff
@@ -250,69 +249,13 @@ def det_common_path_prefix(paths):
         return None
 
 
-def download_file(filename, url, path):
```

Comment:
whel that doesn't work now, because the reason for the moving to a separate module is to solve a circular dependency, so I can't go on importing it back here.